### DISCUSSION TITLE
"style" causing build errors

### DISCUSSION THREAD

**Discussion by theamdrlj** [dicebear/dicebear#167] [Q&A]
Hi, trying to implement dicebear but I got blocked by an error.

This is my code:

```
import { createAvatar } from '@dicebear/avatars'
import * as style from '@dicebear/micah'

let seed
const avatarImage = createAvatar(style, {
  seed,
  dataUri: true
})
```

The error I'm receiving:
```
Argument of type 'typeof import("c:xxMY_REPOxx/node_modules/@dicebear/micah/dist/index")' is not assignable to parameter of type 'Style<Options>'.
  The types of 'schema.enum' are incompatible between these types.
    Type 'import("c:xxMY_REPOxx/node_modules/@types/json-schema/index").JSONSchema7Type[]' is not assignable to type 'import("c:xxMY_REPOxx/node_modules/@dicebear/avatars/node_modules/@types/json-schema/index").JSONSchema7Type[]'.
      Type 'import("c:xxMY_REPOxx/node_modules/@types/json-schema/index").JSONSchema7Type' is not assignable to type 'import("c:xxMY_REPOxx/node_modules/@dicebear/avatars/node_modules/@types/json-schema/index").JSONSchema7Type'.
        Type 'object' is not assignable to type 'JSONSchema7Type'.
          Type '{}' is missing the following properties from type 'JSONSchema7Type[]': length, pop, push, concat, and 28 more.
```

Picture of the build error:
<img width="620" alt="Bug" src="https://user-images.githubusercontent.com/71266422/137150859-98285baf-9b68-47ed-a192-e1b5f044667f.PNG">

Thanks in advance!

**Comment by FlorianKoerner** [comment]
I think there is a problem between the dependencies. What does your package.json look like?

**Reply by theamdrlj** [reply]
Hi, here's the changes in my package.json:

![image](https://user-images.githubusercontent.com/71266422/137574655-1dcb0c02-6bf5-4a47-96e1-75bedcb39b08.png)

**Reply by FlorianKoerner** [reply]
I need the complete package.json. Can't reproduce it at the moment.

**Reply by FlorianKoerner** [reply]
If you'd prefer not to share your package.json, check out this workaround:

https://github.com/dicebear/dicebear/issues/150#issuecomment-910050268

**Comment by theamdrlj** [comment]
Hello.

Thanks for taking time. I have skipLibCheck as true already. I attached my
package.json here, thanks!

Cheers,
Thea
tabbr

**Reply by FlorianKoerner** [reply]
Your package.json was unfortunately not attached.
